migrate.py

```diff
@@ -1974,17 +1974,33 @@ def _rewrite_yaml_lookup(value, namespace, collection, spec, args):
     return value
 
 
+def get_python_module(module_name, module_locations):
+    for module_location in module_locations:
+        module_spec = importlib.util.spec_from_file_location(module_name, module_location)
+        imported_module = importlib.util.module_from_spec(module_spec)
+        try:
```

Comment:
This way you can drop except-block:
```suggestion
        with contextlib.suppress(FileNotFoundError):
```